compiler/rustc_resolve/src/late.rs

```diff
@@ -273,15 +288,17 @@ enum LifetimeRibKind {
     /// Signal we cannot find which should be the anonymous lifetime.
     ElisionFailure,
 
-    /// FIXME(const_generics): This patches over an ICE caused by non-'static lifetimes in const
-    /// generics. We are disallowing this until we can decide on how we want to handle non-'static
-    /// lifetimes in const generics. See issue #74052 for discussion.
-    ConstGeneric,
+    /// This rib forbids usage of generic parameters inside of const parameter types.
+    ///
+    /// See issue #74052 for discussion.
```

Comment:
Pre-existing: should that discussion be summarized here?

Reply:
I think the only really useful information in that issue is a link to rust-lang/project-const-generics#28 which cant be summarized very easily, I'll update this to directly point there instead.